Author: csagan5 <32685696+csagan5@users.noreply.github.com>
Date:   Mon Feb 12 21:30:30 2018 +0100

    ungoogled-chromium: Disable intranet redirect detector
    Disables the intranet redirect detector. It generates extra DNS requests and the functionality using this is disabled
    See this page for more information: https://mikewest.org/2012/02/chrome-connects-to-three-random-domains-at-startup

diff --git a/chrome/browser/intranet_redirect_detector.cc b/chrome/browser/intranet_redirect_detector.cc
--- a/chrome/browser/intranet_redirect_detector.cc
+++ b/chrome/browser/intranet_redirect_detector.cc
@@ -72,9 +72,7 @@ void IntranetRedirectDetector::FinishSleep() {
   fetchers_.clear();
   resulting_origins_.clear();
 
-  const base::CommandLine* cmd_line = base::CommandLine::ForCurrentProcess();
-  if (cmd_line->HasSwitch(switches::kDisableBackgroundNetworking))
-    return;
+  return;
 
   DCHECK(fetchers_.empty() && resulting_origins_.empty());
 
